src/TinyHelpers/Extensions/StringExtensions.cs

```diff
@@ -17,8 +17,15 @@ public static bool EndsWithIgnoreCase(this string? a, string b)
     public static bool ContainsIgnoreCase(this string? a, string b)
         => a?.IndexOf(b, StringComparison.OrdinalIgnoreCase) >= 0;
 
+    /// <summary>
+    /// Returns a new string in which all occurrences of a specified string in the current instance are replaced with another specified string, ignoring the letter casing.
+    /// </summary>
+    /// <param name="input">The original string</param>
+    /// <param name="pattern">The string to be replaced, case insentive.</param>
+    /// <param name="replacement">The string to replace all occurrences of <paramref name="pattern"/> </param>
+    /// <returns></returns>
```

Comment:
You should complete the XML Documentation in the `result ` tag.

Reply:
Sorry, I forgot. Added now the XML documentation for the `returns` tag